src/services/webclient.ts

```diff
@@ -568,9 +569,31 @@ export class WebClientService implements threema.WebClientService {
     }
 
     // Mark a component as initialized
-    public registerInitializationStep(name: string) {
-        this.initialized += 1;
-        if (this.initialized >= this.initializedThreshold) {
+    public registerInitializationStep(name: threema.InitializationStep) {
+        if (this.initialized.indexOf(name) > -1 ) {
+            this.$log.warn('initialization step ', name, ' already registered');
```

Comment:
Spaces are not necessary when concatenating logs with a comma :) And could you add the logTag to the log?

    this.$log.warn(this.logTag, 'Initialization step', name, 'already registered');

Same thing on line 590 and 2300.